### PR TITLE
Fix wrong context panic

### PR DESCRIPTION
**Pull request by liangyuanpeng**
**What happened**:

Pulsarctl panic when use wrong context

```
lanrendeMacBook-Pro:pulsarctl lanren$ go build -o pulsarctl main.go
lanrendeMacBook-Pro:pulsarctl lanren$ ./pulsarctl context set hello
Context "hello" created.
lanrendeMacBook-Pro:pulsarctl lanren$ ./pulsarctl context delete hello
warning: this removed your active context, use "pulsarctl context use" to select a different one
deleted context hello from /Users/lanren/.config/pulsar/config
lanrendeMacBook-Pro:pulsarctl lanren$ ./pulsarctl tenants list
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x14296c5]

goroutine 1 [running]:
github.com/streamnative/pulsarctl/pkg/cmdutils.(*ClusterConfig).Client(0xc0000c2d20, 0x1, 0xf, 0xf)
	/Users/lanren/Documents/repositorys/git/pulsarctl/pkg/cmdutils/config.go:170 +0x315
github.com/streamnative/pulsarctl/pkg/cmdutils.NewPulsarClient(...)
	/Users/lanren/Documents/repositorys/git/pulsarctl/pkg/cmdutils/cmdutils.go:87
github.com/streamnative/pulsarctl/pkg/ctl/tenant.doListTenant(0xc0000c88c0, 0xc00030fcd0, 0x14cdbca)
	/Users/lanren/Documents/repositorys/git/pulsarctl/pkg/ctl/tenant/list.go:68 +0x3a
github.com/streamnative/pulsarctl/pkg/ctl/tenant.listTenantCmd.func1(0x13e6380, 0xc00030fce0)
	/Users/lanren/Documents/repositorys/git/pulsarctl/pkg/ctl/tenant/list.go:61 +0x2a
github.com/streamnative/pulsarctl/pkg/cmdutils.run(0xc0000a1830)
	/Users/lanren/Documents/repositorys/git/pulsarctl/pkg/cmdutils/verb.go:100 +0x27
github.com/streamnative/pulsarctl/pkg/cmdutils.(*VerbCmd).SetRunFunc.func1(0xc000152a00, 0x1a4e7b0, 0x0, 0x0)
	/Users/lanren/Documents/repositorys/git/pulsarctl/pkg/cmdutils/verb.go:67 +0x2a
github.com/spf13/cobra.(*Command).execute(0xc000152a00, 0x1a4e7b0, 0x0, 0x0, 0xc000152a00, 0x1a4e7b0)
	/Users/lanren/Documents/gopath/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:830 +0x2aa
github.com/spf13/cobra.(*Command).ExecuteC(0xc0000d1900, 0xc0002faf40, 0xc0000a2190, 0x2)
	/Users/lanren/Documents/gopath/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914 +0x2fb
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/lanren/Documents/gopath/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
main.main()
	/Users/lanren/Documents/repositorys/git/pulsarctl/main.go:41 +0x1b8
```

**What you expected to happen**:

Just alert error log  

**How to reproduce it (as minimally and precisely as possible)**:  

1. pulsarctl context set hello  
2. pulsarctl context delete hello
3. pulsarctl tenants list

### CODE REVIEW

File: pkg/cmdutils/config.go [hunk 1]
```diff
@@ -160,7 +160,11 @@ func (c *ClusterConfig) Client(version common.APIVersion) pulsar.Client {
 	ctxConf := c.DecodeContext()
 	if ctxConf != nil {
 		if ctxConf.CurrentContext != "" {
-			ctx := ctxConf.Contexts[ctxConf.CurrentContext]
+			ctx, ok := ctxConf.Contexts[ctxConf.CurrentContext]
+			if !ok {
+				logger.Critical("wrong context:%s\n", ctxConf.CurrentContext)
+				os.Exit(1)
+			}
 			auth := ctxConf.AuthInfos[ctxConf.CurrentContext]
```
Comment (zymap) [inline pkg/cmdutils/config.go:168]: Maybe this has the same issue.

**Comment by zymap** [comment]
Thanks @liangyuanpeng for fixing this issue! I left a comment, PTAL.

**Comment by sijie** [comment]
@liangyuanpeng Can you address Yong's comment?

**Comment by liangyuanpeng** [comment]
@sijie  @zymap 

Actually , i have really reply it and ... i didn't note that  status of pending , i am sorry about that. Let we continue to do it!

image

File: pkg/cmdutils/config.go (hunk 1 above)
Comment (sijie) [reply pkg/cmdutils/config.go:168]: I think what @zymap means here is that you need to add a similar code change for `auth := ctxConf.AuthInfos[ctxConf.CurrentContext]`. Can you do the same thing there?

Comment (liangyuanpeng) [reply pkg/cmdutils/config.go:168]: Absolutely :)

**Pull request merged**